client/components/TitleBlock.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <div class="lg:flex lg:items-center lg:justify-between">
     <div class="min-w-0 flex-1">
-      <h2 class="text-2xl font-bold leading-7 text-gray-900 sm:truncate sm:text-3xl sm:tracking-tight">{{ title }}</h2>
+      <h2 class="text-2xl font-bold leading-7 dark:text-neutral-300 sm:truncate sm:text-3xl sm:tracking-tight">{{ title }}</h2>
       <div v-if="summary" class="mt-1 flex flex-col sm:mt-0 sm:flex-row sm:flex-wrap sm:space-x-6">
-        <div class="mt-2 flex items-center text-sm text-gray-500">
+        <div class="mt-2 flex items-center text-sm dark:text-neutral-300">
```

Comment:
Was removing text-gray-900 here intentional?

Reply:
I meant to delete it. I've pushed a better fix for this now in [6d26818](https://github.com/ietf-tools/rpc/pull/150/commits/6d26818241405e383901dc082c1455189bb2944a)